src/packages.builds

```diff
@@ -36,6 +36,10 @@
     <Project Include=".\Native\pkg\runtime.native.System.IO.Compression\runtime.native.System.IO.Compression.builds">
       <AdditionalProperties>$(AdditionalProperties);SkipCreatePackageOnMissingFiles=true</AdditionalProperties>
     </Project>
+    <Project Include=".\Native\pkg\runtime.native.System.Security.Cryptography.Apple\runtime.native.System.Security.Cryptography.Apple.builds">
```

Comment:
Do you know what is supposed to pull this package into the closure? I know that eric also added https://github.com/dotnet/core-setup/commit/2c4e4038a7b8ea70145bc72554c18ff438422d74#diff-d42418420888e3555f05ef5c16792861R45 in core-setup to get it to be pulled in but I believe this also gets referenced by the latest Microsoft.NETCore.Targets package which we are already building due to the compression changes. At any rate is worth doing some basic validation with the packages to make sure it gets correctly pulled in.